src/component/association-select.html

```diff
@@ -1,6 +1,6 @@
 <template>
   <select class="form-control" value.bind="value" multiple.bind="multiple">
-    <option selected disabled.bind="!hidePlaceholder && !selectablePlaceholder" value="0" show.bind="!hidePlaceholder">${placeholderText || '- Select a value -'}</option>
+    <option selected disabled.bind="!hidePlaceholder && !selectablePlaceholder" value="0" show.bind="!hidePlaceholder" t="${placeholderText || '- Select a value -'}">${placeholderText || '- Select a value -'}</option>
```

Comment:
`i18n` will replace the `innerText`, if there is no `i18n` loaded it will use the `innerText`, so no :)